test/helpers/benchmark.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2018 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package helpers
+
+import (
+	"fmt"
+
+	"github.com/cilium/cilium/test/ginkgo-ext"
+)
+
+func (s *SSHMeta) superNetperfRRIPv4(client string, server string, num int) *CmdRes {
```

Comment:
could we move this functions to the benchmark code? Is not really a global helper at all and has some asserts  in there.